.ci/pipeline_definitions

```diff
@@ -0,0 +1,27 @@
+ls-setup-template-image:
```

Comment:
does this already do anything useful ?

Reply:
The idea here is that the ci pipeline is triggered on release of the setup submodule (this project). It will then try to build an image using the dockerfile and push that image into the image repository. It will not actually test/validate the scripts, because this needs a complete environment and will happen when the setup submodule is updated in the landscape-setup-template project.